package.json

```diff
@@ -4,12 +4,14 @@
   "description": "A library that focuses on finding and analyzing the relationships between AsyncAPI Documents too later output consolidated information about the system architecture. Output format should be customizable and available in different formats like uml, mermaid.js and other.",
   "main": "./lib/index.js",
   "scripts": {
-    "test": "echo \"No test specified\"",
+    "test": "npm run test:lib && npm run cover:report",
     "lint": "eslint --max-warnings 0 --config .eslintrc .",
     "get:version": "echo $npm_package_version",
     "get:name": "echo $npm_package_name",
     "generate:assets": "echo \"Nothing to generate\"",
     "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
+    "cover:report": "nyc report --reporter=text --reporter=html",
```

Comment:
yo, have a look at my comment from here about coverage reports https://github.com/asyncapi/java-spring-cloud-stream-template/pull/124#issuecomment-868371800

so please remove it and as a followup you can do coveralls.